packages/php-wasm/universal/src/lib/proxy-file-system.ts

```diff
@@ -34,3 +34,23 @@ export function proxyFileSystem(
 		);
 	}
 }
+
+/**
+ * Answers whether the given path is to a shared filesystem.
+ *
+ * @param sourceOfTruth - The PHP instance that is the source of truth.
+ * @param path - The path to check.
+ * @returns True if the path is to a shared filesystem, false otherwise.
+ */
+export function isPathToSharedFS(sourceOfTruth: PHP, path: string) {
+	// We can't just import the symbol from the library because
+	// Playground CLI is built as ESM and php-wasm-node is built as
+	// CJS and the imported symbols will different in the production build.
+	const __private__symbol = Object.getOwnPropertySymbols(sourceOfTruth)[0];
```

Comment:
Risky – it will break if we ever get another symbol and the index `[0]` changes value. Let's:

1. Filter by `symbol.description === "__private__dont__use"` to ensure we get the right symbol.
2. Add a unit test to protect us from not finding the symbol – this is a potential runtime issue that TypeScript static typing won't protect us from.

Reply:
Also, this is a hole in the entire concept of a private symbol – it shouldn't be available like that 🙈 I'd love to restrict it more via `defineProperty({ enumerable: false })`, but it sounds like it would break this function?